README.md

```diff
@@ -16,7 +16,7 @@
 
 ## 🧪 Built for Experimenting
 
-This tool is meant to accelerate development on [OpenSauced](https://opensauced.pizza) by giving contributors a way to access [Docusaurus 2](https://docusaurus.io/) powered developer documentation available at [docs.opensauced.pizza](https://docs.opensauced.pizza).
+This tool is meant to accelerate development on [OpenSauced](https://opensauced.pizza) by giving contributors a way to access [Docusaurus 2](https://docusaurus.io/) powered developer documentation available at [opensauced.pizza/docs](https://opensauced.pizza/docs).
```

Comment:
I'm double checking. If we want to test this path on localhost, the should the path be `http://localhost:3000/docs`? Because this path leads to 404 as below screenshot. But if it's `http://localhost:3000/`, then please ignore this.

![Screenshot 2024-06-05 192308](https://github.com/open-sauced/docs/assets/45172775/31ca763f-6447-4538-bec3-431567fb185c)

Reply:
It's 3000. I need to write a guide for this as well. The TLDR is this:

- to run locally, you'll go to 3000
- to look at the deploy preview, you'll go to /docs.